app/controllers/organisations_controller.rb

```diff
@@ -0,0 +1,29 @@
+class OrganisationsController < ApplicationController
+  layout 'application'
+
+  def new
+    @organisation = Organisation.new
+    @organisation.agents << @organisation.agents.build
```

Comment:
actually I learned yesterday that build already adds an item to the parent association, so we can simply do `@organisation.agents.build` . I'll commit this change in 5 minutes